Add status about dnf5 in Rawhide (#89)
packit/status: content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md

---

> title: "Fedora Rawhide Testing Blocked on Dnf5"

Reviewer: I think we should either use dnf/dnf5, or DNF/DNF5.

```suggestion
title: "Fedora Rawhide Testing Blocked on dnf5"
```

date: "2023-06-22T09:30:00+02:00"
affected:
  - Testing Farm
resolved: false
section: issue
severity: disrupted
---

> The dnf5 came to rawhide resulting in various of issues
> when running test in Testing Farm on Rawhide.

Reviewer: ```suggestion
dnf has been upgraded to dnf5 in Rawhide, resulting in various issues
when running tests in Testing Farm.
```

> The Testing Farm team is fixing the issues
> and trying to resolve the issues soon.

Reviewer: ```suggestion
The Testing Farm team is working on fixing the issues
and trying to resolve the situation ASAP.
```

> If you want to help, here are the issues in for DNF5 itself:

Reviewer: ```suggestion
If you want to help, here are upstream issues related to this problem:
```

> The Testing Farm team is working on fixing the issues
> and trying to resolve the situation ASAP.
> 
> If you want to help, here are the issues in for DNF5 itself:
> 
> - https://github.com/rpm-software-management/dnf5/issues/549
> - https://github.com/rpm-software-management/dnf5/issues/168

Reviewer 2: ```suggestion
The Testing Farm team together with dnf5 team is working on fixing the issues
and trying to resolve the situation ASAP.

Related DNF5 issues:

- https://github.com/rpm-software-management/dnf5/issues/549
- https://github.com/rpm-software-management/dnf5/issues/168
```

Author: I wanted to suggest people to contribute..;-)

Reviewer: @lachmanfrantisek Could you also include the dnf5 team part from Laura's suggestion?

Reviewer: Ah, never mind :slightly_smiling_face:

Author: I've included the first suggestion, but let the other as is if you don't mind.
